src/SelfService/Infrastructure/Api/ApiResourceFactory.cs

```diff
@@ -450,7 +472,8 @@ public async Task<CapabilityDetailsApiResource> Convert(Capability capability)
                 setCapabilityMetadata: CreateSetMetadataLinkFor(capability),
                 getCapabilityMetadata: CreateGetMetadataLinkFor(capability),
                 getLinkedTeams: GetLinkedTeams(capability),
-                joinCapability: CreateJoinLinkFor(capability)
+                joinCapability: CreateJoinLinkFor(capability),
+                sendInvitations: await CreateSendInvitationsLinkFor(capability)
```

Comment:
similar here to in IAuthorizationService, do we want to standardize the signatures and not await here. Is it necessary?

Reply:
When we create the link we generate `allowedInteractions` which in turn calls the `AuthorizationService`.
As that one is `async` as per the previous comment-answer this one has to be as well.

You mention standardization, but this block of code has plenty of `await` already -- for exactly the same reason. What standard are you looking for here?